Library/ViewModels/PledgeViewModel.swift

```diff
@@ -707,47 +711,34 @@ private func requiresSCA(_ envelope: StripeSCARequiring) -> Bool {
   return envelope.requiresSCAFlow
 }
 
-private func confirmationString(from project: Project, pledgeTotal: String, date: String)
-  -> String {
-  if project.stats.currentCurrency == project.stats.currency {
-    return Strings.If_the_project_reaches_its_funding_goal_you_will_be_charged_on_project_deadline(
-      project_deadline: date
-    )
-  } else {
-    return Strings.If_the_project_reaches_its_funding_goal_you_will_be_charged_total_on_project_deadline(
-      total: pledgeTotal,
-      project_deadline: date
-    )
-  }
-}
-
-private func attributedConfirmationString(with project: Project, pledgeTotal: Double)
-  -> NSAttributedString? {
+private func attributedConfirmationString(with project: Project, pledgeTotal: Double) -> NSAttributedString {
   let date = Format.date(secondsInUTC: project.dates.deadline, template: "MMMM d, yyyy")
   let pledgeTotal = Format.currency(pledgeTotal, country: project.country)
 
-  let fullString = confirmationString(from: project, pledgeTotal: pledgeTotal, date: date)
-
-  let attributedString: NSMutableAttributedString = NSMutableAttributedString(string: fullString)
-  let fullRange = (fullString as NSString).localizedStandardRange(of: fullString)
-  let rangePledgeTotal: NSRange = (fullString as NSString).localizedStandardRange(of: pledgeTotal)
-  let rangeProjectDeadline: NSRange = (fullString as NSString).localizedStandardRange(of: date)
+  let font = UIFont.ksr_caption1()
+  let foregroundColor = UIColor.ksr_text_dark_grey_500
 
   let paragraphStyle = NSMutableParagraphStyle()
   paragraphStyle.alignment = .center
 
-  let regularFontAttribute = [
-    NSAttributedString.Key.paragraphStyle: paragraphStyle,
-    NSAttributedString.Key.font: UIFont.ksr_caption1(),
-    NSAttributedString.Key.foregroundColor: UIColor.ksr_text_dark_grey_500
+  let attributes = [
+    NSAttributedString.Key.paragraphStyle: paragraphStyle
   ]
-  let boldFontAttribute = [NSAttributedString.Key.font: UIFont.ksr_caption1().bolded]
 
-  attributedString.addAttributes(regularFontAttribute, range: fullRange)
-  attributedString.addAttributes(boldFontAttribute, range: rangePledgeTotal)
-  attributedString.addAttributes(boldFontAttribute, range: rangeProjectDeadline)
+  if project.stats.currentCurrency == project.stats.currency {
```

Comment:
Here too we could use `project.stats.needsConversion`

Reply:
This change seemed to break snapshots. Will dig in more tomorrow.